CHANGELOG.md

```diff
@@ -45,6 +46,7 @@ list only contains the Pylint checkers enabled by default in PythonTA.
 - Fixed bug where `check_errors` and `check_all` opens a webpage when a nonexistent or unreadable path is passed as an argument.
 - Fixed the CFG implementation to resolve a bug in the possibly-undefined checker where variables were falsely flagged as possibly undefined when the code conditionally raises an exception and the variable was referenced afterwards.
 - Fixed bug where the generated CFGs will highlight the except block as unreachable if the same exception it is handling was raised in the body of the tryexcept.
+- Fixed bug where the HTML reporter would display all error occurrences of the same type despite stating that only a limited number was being shown.
```

Comment:
You added this line in the incorrect place, it's under `[2.5.0]`. You can instead add a new `Bug fixes` section under `[unreleased]`.